provision/initramfs/init

```diff
@@ -351,7 +351,7 @@ if [ -n "$WWPOSTSHELL" -a "$WWPOSTSHELL" != "0" ]; then
     setsid /bin/cttyhack /bin/sh
 fi
 
-if [ -x "$NEWROOT/sbin/init" ]; then
+if [ -x "$NEWROOT/sbin/init" ] || [ -h "$NEWROOT/sbin/init" -a -x "$NEWROOT`readlink $NEWROOT/sbin/init`" ]; then
```

Comment:
@gmkurtzer I would rather test if the target is existing and executable.

@jmstover If you do the workaround I mentioned in #2, neither `provision/initramfs/init` nor `provision/initramfs/capabilities/transport-http/wwgetvnfs` need to be chanhged. However I prefer the solution in this pull request, as it does not require changing the chroot. Even if we change the link in the `post_chroot()`, the user may e.g. update the packages in the chroot after some time, and the relative link may be restored back to the absolute one which will prevent booting.